app/presenters/further_information_presentable.rb

```diff
@@ -0,0 +1,40 @@
+module FurtherInformationPresentable
```

Comment:
maybe we could keep this in a `concerns` folder under presenters just it as a potential bit of re-usable code